tests/functional/password_visibility.js

```diff
@@ -71,44 +77,12 @@ define([
         .end();
     },
 
-    'toggle show password for Sync': function () {
-      this.get('remote').get(require.toUrl(SYNC_SIGNIN_URL))
+    'Password manager for Sync': function () {
+      return FunctionalHelpers.openPage(this, SYNC_SIGNIN_URL)
         .findByCssSelector('#fxa-signin-header')
         .end()
 
-        .findByCssSelector('#password')
-          .click()
-          .type('password')
-        .end()
-
-        // turn it into a text field
-        .findByCssSelector('.show-password-label')
-          .click()
-        .end()
-
-        .findByCssSelector('#password')
-          .getAttribute('type')
-          .then(function (typeValue) {
-            assert.equal(typeValue, 'text');
-          })
-
-          .getAttribute('autocomplete')
-          .then(function (autocompleteValue) {
-            assert.equal(autocompleteValue, 'off');
-          })
-        .end()
```

Comment:
Is this "making it visible disables autocomplete" functionality now tested elsewhere?